etc/profile-a-l/discord-common.profile

```diff
@@ -23,7 +23,7 @@ ignore novideo
 whitelist ${HOME}/.config/BetterDiscord
 whitelist ${HOME}/.local/share/betterdiscordctl
 
-private-bin bash,cut,echo,egrep,electron,electron[0-9],electron[0-9][0-9],fish,grep,head,sed,sh,tclsh,tr,xdg-mime,xdg-open,zsh
```

Comment:
do you know why electron was in there before? are there users running something like `firejail --profile=discord.profile electron /opt/discord/...` ?

Reply:
I _assume_ it's due to the way arch linux (and possibly other distro's) packages electron to support having different versions installed side-by-side. The latest electron package always uses the shell script as /usr/bin/electron, while older electrons install it as a versioned /usr/bin/electronX.

Reply:
@reinerh [on Sep 14](https://github.com/netblue30/firejail/pull/5368#discussion_r970497695):

> do you know why electron was in there before? are there users running
> something like `firejail --profile=discord.profile electron /opt/discord/...`
> ?

For reference, it was added on commit ed1f0c088 ("Add electron[0-9]{0,2} to all
electron redirect …", 2021-08-01) by @rusty-snake.

Presumably, electron-based programs need to execute the electron binary
directly.